pkg/sql/show_create.go

```diff
@@ -37,6 +37,16 @@ const (
 	OmitMissingFKClausesFromCreate
 )
 
+type ShowCreateDisplayOptions struct {
+	FKDisplayMode shouldOmitFKClausesFromCreate
+	// Comment resolution requires looking up table data from system.comments
+	// table. This is sometimes not possible. For example, in the context of a
+	// SHOW BACKUP which may resolve the create statement, there is no mechanism
+	// to read any table data from the backup (nor is there a guarantee that the
+	// system.comments table is included in the backup at all).
+	IgnoreComments bool
```

Comment:
I went back and forth on the naming of this a few times. This really is just mandating that you don't want the show statement to escape past the lookup context provided. I ended up going for the most direct name, but am curious if you have any thoughts about this.

Reply:
cc @knz on this one since i don't have thoughts here.

We could just go back to calling the free function `sql.ShowCreate` instead though, since it did what we want (until we fix it for real) before it was switched to this planner method, right?

Reply:
I think both approaches are equally correct here. I'd recommend the one that you think will minimize the maintenance burden.

Reply:
The free function was changed to this planner func. I went this route in order to avoid maintaining 2 versions of ShowCreate.